tests/acceptance/editions/octane-test.js

```diff
@@ -25,16 +24,6 @@ module('Acceptance | editions/octane', function (hooks) {
 
   test('Accessibility audit', async function (assert) {
     await visit('/editions/octane');
-    await a11yAudit({
-      rules: {
-        'aria-valid-attr-value': {
-          enabled: false,
-        },
-        'heading-order': {
-          enabled: false,
-        },
-      },
-    });
```

Comment:
Ah, I wasn't clear in my issue description, sorry. After addressing the 2 accessibility issues, we want to update lines 28-37 to:

```javascript
await a11yAudit();
```

You can visit [ember-a11y-testing documentation page](https://github.com/ember-a11y/ember-a11y-testing#setting-options-using-setrunoptions) to learn more about the JavaScript object that was previously passed.

In essence, passing an object allows us to temporarily ignore accessibility issues. This can be useful when updating a large app, and reminds us that we should address the issues at some point. :)

Could you revert the deletion and add back the import statement in line 3?